apps/api/src/app/environments-v1/usecases/outputRendererUseCases.ts

```diff
@@ -0,0 +1,144 @@
+// Base interface for all renderers
```

Comment:
Please rename to `output-renderer-usecase.ts` to align the naming with the rest of the use case file. Or use one file for each step type so that its easier to find these files in the future.